tsconfig.json

```diff
@@ -37,16 +37,22 @@
   "exclude": [
     "dist",
     "packages/neo-one-server-plugin-wallet/src/contracts/*.ts",
+    "packages/neo-one-smart-contract-lib/src/access/roles/*.ts",
```

Comment:
I think you should just use
```
"packages/neo-one-smart-contract-lib/src/access/**/*.ts,
```
instead of
```
"packages/neo-one-smart-contract-lib/src/access/*.ts",
"packages/neo-one-smart-contract-lib/src/access/roles/*.ts"
```

could probably do the same for lifecycles.

Reply:
done